draft-ietf-modpod-group-processes.md

```diff
@@ -302,6 +302,9 @@ bans longer than fourteen (14) days SHALL be reported to the forum in
 which the person was banned, and in the case of a ban that spans more
 than one forum, to the community in a manner decided by the IESG.
 
+Moderator actions MAY NOT include content removal or redcaction from
+IETF archives, which are both beyond the scope of this memo.
```

Comment:
```suggestion
Content removal or redaction from IETF archives are not moderation actions;
the IESG MUST initiate and approve any such operation beyond the scope
of this memo.
```

MAY NOT is not a 2119 term.

Reply:
I take your point.  I don't think we want the "IESG MUST..."  thing either, tho.  Let me give it another whack.

Reply:
Just to add, I don't think we want to mention who has responsibility.  Future RFCs can decide that.  We don't need to go there.

Reply:
This is already under the auspices of the IESG per their "management of the standards process" responsibility, no future RFc needed.

Reply:
No what I mean is that we shouldn't discuss who has responsibility for things outside the scope of the document.  For instance, If a later RFC deals with redactions and deletions, they shouldn't have to worry about updating this one.  And I proposed fewer words ;-)

Reply:
How about

"Permissable moderator actions do not include content removal or redcaction from IETF archives, which are both beyond the scope of this memo."

#84 is still open, so that maybe we can leave the CAPS for that issue.

Reply:
SGTM